zerver/lib/events.py

```diff
@@ -595,6 +596,9 @@ def apply_event(
                     state["can_create_streams"] = user_profile.can_create_streams()
                     state["can_subscribe_other_users"] = user_profile.can_subscribe_other_users()
                     state["can_invite_others_to_realm"] = user_profile.can_invite_others_to_realm()
+                    state[
+                        "can_move_messages_to_stream"
```

Comment:
I think `can_move_messages_between_streams` would be a fine name for this.

I think there's also a question of whether we think we should be adding all of these policy things to the `register` response -- given that a correct client needs to dynamically compute this anyway since the role can change as can the policy.

@gnprice @neiljp for your thoughts; do you find these `can_` properties useful?